README.md

```diff
@@ -76,9 +86,14 @@ generate_measures:
       measure: output/joined/measure_*.csv
 ```
 
-[1]: https://github.com/opensafely-actions/cohort-joiner/tags
-[cohort-extractor]: https://docs.opensafely.org/actions-cohortextractor/
-
 ## Notes for developers
 
 Please see [_DEVELOPERS.md_](DEVELOPERS.md).
+
+[1]: https://github.com/opensafely-actions/cohort-joiner/tags
```

Comment:
ohh this way of creating links is soo nice :) I didnt know this works, will use this too in future

Reply:
It's tidier than inline links, when they are long. More at https://commonmark.org/help/.